clients/mtaclient/mta_client_operations.go

```diff
@@ -11,14 +11,14 @@ import (
 type MtaClientOperations interface {
 	ExecuteAction(operationID, actionID string) (ResponseHeader, error)
 	GetMta(mtaID string) (*models.Mta, error)
-	GetMtaFiles() ([]*models.FileMetadata, error)
+	GetMtaFiles(namespace *string) ([]*models.FileMetadata, error)
```

Comment:
In Go, and in C++, it is a practise(not that good) that you can provide pointer to an object and later on change the value of the pointer in the calling function. This could be useful in some cases but I think that in your case it would be better to pass the variable by value. I suppose you do not want to change it in the method, right? :)

Reply:
This was done since the namespace is an optional parameter and can be an empty reference. A similar thing is done below for 'GetMtaOperations(mtaId *string...'